### PR TITLE
fix: log arguments with debug

### PR DESCRIPTION
**Pull request by mr-cal**
- [x] Have you followed the guidelines for contributing?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] Have you successfully run `tox`?

Parsed arguments can be helpful when debugging.

### CODE REVIEW

File: craft_cli/dispatcher.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def load_command(self, app_config: Any) -> BaseCommand:
         parser = _CustomArgumentParser(self._help_builder, prog=self._loaded_command.name)
         self._loaded_command.fill_parser(parser)
         self._parsed_command_args = parser.parse_args(self._command_args)
-        emit.trace(f"Command parsed sysargs: {self._parsed_command_args}")
+        emit.debug(f"Command parsed sysargs: {self._parsed_command_args}")
```
Comment (tigarmo) [inline craft_cli/dispatcher.py:248]: I know that I was the one that suggested this but now I'm afraid of leaking stuff that we shouldn't leak from the command line. For example, what if someone calls `snapcraft pack  --ua-token <token>`?

Comment (lengau) [reply craft_cli/dispatcher.py:248]: This is a general problem with CLI stuff. It gets logged in your bash history too.

Secrets like that really should be passed through environment variables.

**Review by lengau — APPROVED** [review]
Wouldn't this be increasing the logging rather than reducing? 🤓

**Comment by mr-cal** [comment]
> Wouldn't this be increasing the logging rather than reducing? 🤓

Ha yes.  I was thinking reducing the log level, which does increase the amount of logging.

**Comment by mr-cal** [comment]
As discussed with @tigarmo and @lengau, we need a solution to hide secrets before logging command line arguements.

**Pull request closed without merging**